src/main/java/umaikaze/duke/TaskList.java

```diff
@@ -0,0 +1,97 @@
+package umaikaze.duke;
+
+import umaikaze.duke.task.Deadline;
+import umaikaze.duke.task.Event;
+import umaikaze.duke.task.Task;
+import umaikaze.duke.task.Todo;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    List<Task> list;
+
+    public TaskList() {
+        list = new ArrayList<>(100);
+    }
+
+    public TaskList(List<Task> loaded) {
+        list = loaded;
+    }
+
+    public String countList() {
+        return "Nyow you have " + list.size() + " tasks in the wist.";
+    }
+
+    private Task getTask(String[] line) throws DukeException{
+        Task newTask;
+        String cmd = line[0];
+        Parser p = new Parser(line);
+        switch (cmd) {
+            case "deadline":
+                if (p.getDescription().equals("")) {
+                    throw new DukeException("OOPS ;;ω;;  The descwiption of a deadwinye cannyot be empty.");
```

Comment:
Should `throw new DukeException` be considered user interaction? If so, is it possible to let your `ui.class` to implement a method called `showUserError()` so that you can call `ui` directly here?